src/react-components/ui-root.js

```diff
@@ -814,6 +819,10 @@ class UIRoot extends Component {
           </div>
 
           <img className="loading-panel__logo" src="../assets/images/hub-preview-light-no-shadow.png" />
+
+          <h4 className={this.state.loadingText.indexOf("lobby") !== -1 ? "loadedText" : "loadingText"}>
```

Comment:
Bit weird for this conditional to be based on the contents of the loading text. Would be better to add a boolean state field instead.